### PR TITLE
feat(UH-013-014): create tracking table

### PR DESCRIPTION
**Pull request by TheHoodini**
Tracking Table:

image

### CODE REVIEW

File: src/features/tracking/pages/TrackingList/TrackingTable.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            icon={<EditIcon />}
+            label="Edit"
+            className="textPrimary"
+            component={Link}
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            to={`/monitoring/update/${id}`}
+            color="inherit"
+          />,
+          <GridActionsCellItem
```
Comment (dgop92) [inline src/features/tracking/pages/TrackingList/TrackingTable.tsx]: a backend level is called monitoring, but at client livel is tracking

**Review by dgop92 — CHANGES_REQUESTED** [review]
review the small changes

**Review by AlejandroSalas1143 — CHANGES_REQUESTED** [review]
The table is not where it should go. Please position it according to what the PDF says.

**Review by dgop92 — APPROVED** [review]
ok

**Review by ndelarans — APPROVED** [review]
Ok

**Review by AlejandroSalas1143 — APPROVED** [review]
good

**Pull request merged**